# Cebuano translation for apport
# Copyright (c) 2018 Rosetta Contributors and Canonical Ltd 2018
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-06-06 11:55+0200\n"
"PO-Revision-Date: 2018-08-22 11:00+0000\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: Cebuano <ceb@li.org>\n"
"Language: ceb\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-08-19 12:20+0000\n"
"X-Generator: Launchpad (build 87919ae1fbf1142b0e02b80650e6ccf3e775beb3)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""

#: ../apport/ui.py:263
msgid "This package does not seem to be installed correctly"
msgstr ""

#: ../apport/ui.py:273
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:303
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""

#: ../apport/ui.py:454
msgid "unknown program"
msgstr ""

#: ../apport/ui.py:456
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr ""

#: ../apport/ui.py:458
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""

#: ../apport/ui.py:463 ../apport/ui.py:1872
#, python-format
msgid "Problem in %s"
msgstr ""

#: ../apport/ui.py:519 ../apport/ui.py:536 ../apport/ui.py:695
#: ../apport/ui.py:701 ../apport/ui.py:931 ../apport/ui.py:1695
#: ../apport/ui.py:1839 ../apport/ui.py:1845
msgid "Invalid problem report"
msgstr ""

#: ../apport/ui.py:520
msgid "You are not allowed to access this problem report."
msgstr ""

#: ../apport/ui.py:528
msgid "Error"
msgstr ""

#: ../apport/ui.py:530
msgid "There is not enough disk space available to process this report."
msgstr ""

#: ../apport/ui.py:569 ../apport/ui.py:674
msgid "Invalid PID"
msgstr ""

#: ../apport/ui.py:569
msgid "The specified process ID does not exist."
msgstr ""

#: ../apport/ui.py:574
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:575
msgid "The specified process ID does not belong to you."
msgstr ""

#: ../apport/ui.py:632
msgid "No package specified"
msgstr ""

#: ../apport/ui.py:634
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:661
msgid "Permission denied"
msgstr ""

#: ../apport/ui.py:663
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""

#: ../apport/ui.py:675
msgid "The specified process ID does not belong to a program."
msgstr ""

#: ../apport/ui.py:696
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr ""

#: ../apport/ui.py:702
#, python-format
msgid "Package %s does not exist"
msgstr ""

#: ../apport/ui.py:731 ../apport/ui.py:936 ../apport/ui.py:969
#: ../apport/ui.py:976
msgid "Cannot create report"
msgstr ""

#: ../apport/ui.py:748 ../apport/ui.py:802 ../apport/ui.py:819
msgid "Updating problem report"
msgstr ""

#: ../apport/ui.py:750
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""

#: ../apport/ui.py:762
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""

#: ../apport/ui.py:802 ../apport/ui.py:819
msgid "No additional information collected."
msgstr ""

#: ../apport/ui.py:875
msgid "What kind of problem do you want to report?"
msgstr ""

#: ../apport/ui.py:894
msgid "Unknown symptom"
msgstr ""

#: ../apport/ui.py:895
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr ""

#: ../apport/ui.py:915
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:916
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:938
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:954
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""

#: ../apport/ui.py:970 ../apport/ui.py:977
msgid "xprop failed to determine process ID of the window"
msgstr ""

#: ../apport/ui.py:992
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:993
msgid "Specify package name."
msgstr ""

#: ../apport/ui.py:999 ../apport/ui.py:1127
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""

#: ../apport/ui.py:1035
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:1046
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""

#: ../apport/ui.py:1055
msgid "Click a window as a target for filing a problem report."
msgstr ""

#: ../apport/ui.py:1063
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""

#: ../apport/ui.py:1071
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1080
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""

#: ../apport/ui.py:1091
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1099
msgid "The provided pid is a hanging application."
msgstr ""

#: ../apport/ui.py:1107
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""

#: ../apport/ui.py:1117
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:1133
msgid "Print the Apport version number."
msgstr ""

#: ../apport/ui.py:1292
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""

#: ../apport/ui.py:1296
msgid "Run gdb session"
msgstr ""

#: ../apport/ui.py:1297
msgid "Run gdb session without downloading debug symbols"
msgstr ""

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1299
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr ""

#: ../apport/ui.py:1353
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1357
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1439 ../apport/ui.py:1452
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""

#: ../apport/ui.py:1472
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""

#. can happen with broken core dumps
#. can happen with broken gz values
#: ../apport/ui.py:1531 ../apport/ui.py:1643 ../apport/ui.py:1843
msgid "This problem report is damaged and cannot be processed."
msgstr ""

#: ../apport/ui.py:1537
msgid "This report is about a package that is not installed."
msgstr ""

#: ../apport/ui.py:1544
msgid "An error occurred while attempting to process this problem report:"
msgstr ""

#: ../apport/ui.py:1561
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1566
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1567
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1605
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1610
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1696
msgid "Could not determine the package or source package name."
msgstr ""

#: ../apport/ui.py:1727
msgid "Unable to start web browser"
msgstr ""

#: ../apport/ui.py:1728
#, python-format
msgid "Unable to start web browser to open %s."
msgstr ""

#: ../apport/ui.py:1801
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""

#: ../apport/ui.py:1804
msgid "Network problem"
msgstr ""

#: ../apport/ui.py:1830
msgid "Memory exhaustion"
msgstr ""

#: ../apport/ui.py:1832
msgid "Your system does not have enough memory to process this crash report."
msgstr ""

#: ../apport/ui.py:1877
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""

#: ../apport/ui.py:1931 ../apport/ui.py:1943
msgid "Problem already known"
msgstr ""

#: ../apport/ui.py:1933
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""

#: ../apport/ui.py:1944
msgid "This problem was already reported to developers. Thank you!"
msgstr ""

#: ../bin/apport-cli.py:90
msgid "Press any key to continue..."
msgstr ""

#: ../bin/apport-cli.py:97
msgid "What would you like to do? Your options are:"
msgstr ""

#: ../bin/apport-cli.py:110
#, python-format
msgid "Please choose (%s):"
msgstr "Palihug pamili (%s):"

#: ../bin/apport-cli.py:170
#, python-format
msgid "(%i bytes)"
msgstr ""

#: ../bin/apport-cli.py:172 ../gtk/apport-gtk.py:136 ../kde/apport-kde.py:417
msgid "(binary data)"
msgstr "(binary data)"

#: ../bin/apport-cli.py:212 ../gtk/apport-gtk.py:176 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:208
msgid "Send problem report to the developers?"
msgstr ""

#: ../bin/apport-cli.py:214
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""

#: ../bin/apport-cli.py:221
#, python-format
msgid "&Send report (%s)"
msgstr ""

#: ../bin/apport-cli.py:225
msgid "&Examine locally"
msgstr ""

#: ../bin/apport-cli.py:229
msgid "&View report"
msgstr ""

#: ../bin/apport-cli.py:231
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""

#: ../bin/apport-cli.py:234
msgid "Cancel and &ignore future crashes of this program version"
msgstr ""

#: ../bin/apport-cli.py:237 ../bin/apport-cli.py:317 ../bin/apport-cli.py:353
#: ../bin/apport-cli.py:374
msgid "&Cancel"
msgstr "&Kanselahon"

#: ../bin/apport-cli.py:258
msgid "Problem report file:"
msgstr ""

#: ../bin/apport-cli.py:264 ../bin/apport-cli.py:269
msgid "&Confirm"
msgstr ""

#: ../bin/apport-cli.py:268
#, python-format
msgid "Error: %s"
msgstr "Sayup: %s"

#: ../bin/apport-cli.py:274 ../kde/apport-kde.py:465
msgid "Collecting problem information"
msgstr ""

#: ../bin/apport-cli.py:276
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""

#: ../bin/apport-cli.py:292 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:516
msgid "Uploading problem information"
msgstr ""

#: ../bin/apport-cli.py:294
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""

#: ../bin/apport-cli.py:315
msgid "&Yes"
msgstr ""

#: ../bin/apport-cli.py:316
msgid "&No"
msgstr ""

#: ../bin/apport-cli.py:352
msgid "&Done"
msgstr "&Humana"

#: ../bin/apport-cli.py:358
msgid "none"
msgstr "wala"

#: ../bin/apport-cli.py:359
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr ""

#: ../bin/apport-cli.py:375
msgid "Choices:"
msgstr "Pili-anan:"

#: ../bin/apport-cli.py:389
msgid "Path to file (Enter to cancel):"
msgstr ""

#: ../bin/apport-cli.py:395
msgid "File does not exist."
msgstr ""

#: ../bin/apport-cli.py:397
msgid "This is a directory."
msgstr ""

#: ../bin/apport-cli.py:402
msgid "To continue, you must visit the following URL:"
msgstr ""

#: ../bin/apport-cli.py:404
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""

#: ../bin/apport-cli.py:409
msgid "Launch a browser now"
msgstr ""

#: ../bin/apport-cli.py:424
msgid "No pending crash reports. Try --help for more information."
msgstr ""

#: ../bin/apport-retrace.py:48
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""

#: ../bin/apport-retrace.py:55
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""

#: ../bin/apport-retrace.py:64
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""

#: ../bin/apport-retrace.py:73
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""

#: ../bin/apport-retrace.py:76
msgid "Override report's CoreFile"
msgstr ""

#: ../bin/apport-retrace.py:79
msgid "Override report's ExecutablePath"
msgstr ""

#: ../bin/apport-retrace.py:82
msgid "Override report's ProcMaps"
msgstr ""

#: ../bin/apport-retrace.py:88
msgid "Rebuild report's Package information"
msgstr ""

#: ../bin/apport-retrace.py:95
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""

#: ../bin/apport-retrace.py:109
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""

#: ../bin/apport-retrace.py:119
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-retrace.py:125
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""

#: ../bin/apport-retrace.py:131
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:139
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../bin/apport-retrace.py:145
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""

#: ../bin/apport-retrace.py:156 ../bin/apport-valgrind.py:102
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""

#: ../bin/apport-retrace.py:163
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""

#: ../bin/apport-retrace.py:173
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""

#: ../bin/apport-retrace.py:181
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""

#: ../bin/apport-retrace.py:188
msgid "Do not add StacktraceSource to the report."
msgstr ""

#: ../bin/apport-retrace.py:201
msgid "You cannot use -C without -S. Stopping."
msgstr ""

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:231
msgid "OK to send these as attachments? [y/n]"
msgstr ""

#: ../bin/apport-unpack.py:35
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:36
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:37
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:86
msgid "Destination directory exists and is not empty."
msgstr ""

#: ../bin/apport-valgrind.py:43
msgid "See man page for details."
msgstr ""

#: ../bin/apport-valgrind.py:52
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../bin/apport-valgrind.py:58
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""

#: ../bin/apport-valgrind.py:66
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""

#: ../bin/apport-valgrind.py:75
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""

#: ../bin/apport-valgrind.py:84
msgid "report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-valgrind.py:91
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:133
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr ""

#: ../data/apportcheckresume.py:75
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:80
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:88
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""

#: ../data/kernel_oops.py:34
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr ""

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr ""

#: ../gtk/apport-gtk.py:153
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:156
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:161 ../kde/apport-kde.py:223 ../kde/apport-kde.py:272
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr ""

#: ../gtk/apport-gtk.py:184 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:216
msgid "Send"
msgstr "Ipadala"

#: ../gtk/apport-gtk.py:206 ../gtk/apport-gtk.py:620 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:333
msgid "Show Details"
msgstr "Ipakita ang mga detalye"

#: ../gtk/apport-gtk.py:219 ../kde/apport-kde.py:264 ../kde/apport-kde.py:279
msgid "Continue"
msgstr "Padayon"

#: ../gtk/apport-gtk.py:243
#, python-format
msgid "The application %s has stopped responding."
msgstr ""

#: ../gtk/apport-gtk.py:247
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr ""

#: ../gtk/apport-gtk.py:264 ../kde/apport-kde.py:231
#, python-format
msgid "Package: %s"
msgstr "Pakete: %s"

#: ../gtk/apport-gtk.py:270 ../kde/apport-kde.py:238
msgid "Sorry, a problem occurred while installing software."
msgstr ""

#: ../gtk/apport-gtk.py:280 ../gtk/apport-gtk.py:299 ../kde/apport-kde.py:246
#, python-format
msgid "The application %s has experienced an internal error."
msgstr ""

#: ../gtk/apport-gtk.py:282 ../kde/apport-kde.py:251
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:307 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:276
msgid "If you notice further problems, try restarting the computer."
msgstr ""

#: ../gtk/apport-gtk.py:310 ../kde/apport-kde.py:281
msgid "Ignore future problems of this type"
msgstr ""

#: ../gtk/apport-gtk.py:624 ../kde/apport-kde.py:330
msgid "Hide Details"
msgstr "Itago ang mga Detalye"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:384 ../kde/apport-kde.py:558
msgid "Apport"
msgstr ""

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr ""

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr ""

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr ""

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr ""

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr ""

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr ""

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr ""

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr ""

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr ""

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr ""

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr ""

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:518
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr ""

#: ../kde/apport-kde.py:267
msgid "Leave Closed"
msgstr ""

#: ../kde/apport-kde.py:268 ../kde/apport-kde.py:435
msgid "Relaunch"
msgstr ""

#: ../kde/apport-kde.py:361
msgid "Username:"
msgstr ""

#: ../kde/apport-kde.py:364
msgid "Password:"
msgstr ""

#: ../kde/apport-kde.py:464
msgid "Collecting Problem Information"
msgstr ""

#: ../kde/apport-kde.py:467
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""

#: ../kde/apport-kde.py:515
msgid "Uploading Problem Information"
msgstr ""
